src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -396,6 +396,10 @@
     "NAME": "Scorching Ray",
     "DESCRIPTION": "Remember stsjorbsmod:Lust. NL Apply !stsjorbsmod:MetaMagic! stsjorbsmod:Burning !M! times. This stsjorbsmod:Burning is affected by Strength."
   },
+  "stsjorbsmod:Shake": {
+    "NAME": "Shake",
+    "DESCRIPTION": "If enemy is Vulnerable, draw !M! cards. NL Apply 1 Vulnerable. NL Exhaust."
```

Comment:
You can use placeholders for the vulnerable amount too, it just looks a little different because it's a custom placeholder babe. I think the syntax is `!stsjorbsmod:UrMagic!`

Reply:
It's good to do this even if it isn't changing on upgrade right now because it means any future balance changes to the card mechanics are less likely to require updates for all localization files